packages/main/src/TableNavigation.ts

```diff
@@ -59,7 +59,7 @@ class TableNavigation extends TableExtension {
 			items.push(this._getNavigationItemsOfRow(this._table._nodataRow));
 		}
 
-		if (this._table._shouldRenderGrowing) {
+		if (this._table.rows.length > 0 && this._table._growing?.hasGrowingComponent()) {
```

Comment:
I really didn't mean that but ok

Reply:
Yep, completely misunderstood it. I guess it's time for the weekend
Changed it to _getGrowing similar to the other features
```ts
_getGrowing(): TableGrowing | undefined {
  return this._findFeature<TableGrowing>("TableGrowing");
}
```